File: 백준/Bronze/3003. 킹， 퀸， 룩， 비숍， 나이트， 폰/킹， 퀸， 룩， 비숍， 나이트， 폰.py
import sys
readline = sys.stdin.readline
pices = list(map(int, readline().split()))
sample = [1, 1, 2, 2, 2, 8]


for i in range(6):
    if sample[i] >= pices[i]:
        print(sample[i] - pices[i], end=" ")
    else:
        print((pices[i] - sample[i]) * -1, end=" ")

File: 백준/Bronze/3003. 킹， 퀸， 룩， 비숍， 나이트， 폰/킹， 퀸， 룩， 비숍， 나이트， 폰.swift
import Foundation

let pices = readLine()!.split { $0 == " " }.map { Int($0)! }
let sample = [1, 1, 2, 2, 2, 8]
for i in 0 ..< 6 {
    if sample[i] >= pices[i] {
        print(sample[i] - pices[i], terminator: " ")
    } else {
        print((pices[i] - sample[i]) * -1, terminator: " ")
    }
}
